src/main/java/com/laserfiche/repository/api/clients/impl/EntriesClientImpl.java

```diff
@@ -353,10 +410,15 @@ public CompletableFuture<File> exportDocumentWithAuditReason(String repoId, Inte
     public CompletableFuture<File> exportDocument(String repoId, Integer entryId, String range) {
         Map<String, Object> pathParameters = getNonNullParameters(new String[]{"repoId", "entryId"},
                 new Object[]{repoId, entryId});
+        Map<String, Object> headerParameters = getNonNullParameters(new String[]{"range"}, new Object[]{range});
+        Map<String, String> headerParametersWithStringTypeValue = headerParameters
+                .entrySet()
+                .stream()
+                .collect(Collectors.toMap(Map.Entry::getKey, e -> (String) e.getValue()));
```

Comment:
This seems really nice. I know it's not the job of this PR, but maybe we could have story that convert the other two use cases (in header and in path) to use this instead.

Reply:
I created this user story https://tfs/defaultcollection/Server%20Tools/_workitems/edit/397778 for this.